test/polyrith.lean

```diff
@@ -154,28 +154,28 @@ by test_polyrith "{\"data\":[\"(poly.const 1/1)\"],\"success\":true}" ["ff", "in
 
 example («¥» y : ℤ) (h1 : 3*«¥» + 2*y = 10):
   «¥» * (3*«¥» + 2*y) = 10 * «¥» :=
-by test_polyrith "{\"data\":[\"(poly.mul (poly.const 1/1) (poly.var 0))\"],\"success\":true}" ["ff", "int", "2", "[(((3 * var0) + (2 * var1)) - 10)]", "((var0 * ((3 * var0) + (2 * var1))) - (10 * var0))"]  "linear_combination 1 * «¥» * h1"
+by test_polyrith "{\"data\":[\"(poly.var 0)\"],\"success\":true}" ["ff", "int", "2", "[(((3 * var0) + (2 * var1)) - 10)]", "((var0 * ((3 * var0) + (2 * var1))) - (10 * var0))"]  "linear_combination «¥» * h1"
```

Comment:
For the sake of a clearer diff; could you spin off a PR that that puts each of the arguments to `test_polyrith` on a separate line? It would make this (and future polyrith PRs) easier to review, especially for the really long example below.

Reply:
I can, but not immediately. Updating these tests will get a lot easier if/when https://github.com/leanprover/vscode-lean/issues/308 is fixed.

Reply:
@eric-wieser making this change is easy enough. I can't bear the thought right now of spinning off that PR, updating every test, and then immediately rebasing this one and updating every test again. 

Do you think it's really necessary? I'll happily PR the newline fix for future use.

Reply:
Would your opinion change given I've pushed a fix for the vscode-lean extension?